experiments/spotify/1_no_code_consume_source/README.md

```diff
@@ -0,0 +1,36 @@
+# Consume a ready source without Python code
+We can load a source from the command line. Imagine this:
+
+```
+pip install python-dlt
+pip install python-dlt-contrib
+
+dlt init
+
+vi .dlt/secrets
+
+dlt source info dlt.contrib.sources.spotify
+
+> displays the source docstring and parameters to be passed - all from function signature!
+
+dlt load --to bigquery --dataset spotify dlt.contrib.sources.spotify --data-dir my_data --tables library,history
```

Comment:
how does this look if you are using the Spotify API instead of a data directory with JSON files from Spotify?